src/Microsoft.Identity.Web/DownstreamWebApiSupport/DownstreamWebApi.cs

```diff
@@ -74,6 +76,16 @@ public class DownstreamWebApi : IDownstreamWebApi
                 userflow = effectiveOptions.UserFlow;
             }
 
+            if (cancellationToken != default)
```

Comment:
I don't think this check is needed. Passing along a default cancellation token is ok.